instaloader/instaloader.py

```diff
@@ -517,6 +539,9 @@ def download_post(self, post: Post, target: Union[str, Path]) -> bool:
             if post.typename == 'GraphSidecar':
                 edge_number = 1
                 for sidecar_node in post.get_sidecar_nodes():
+                    if self.slide_start + self.slide_end != 0:
+                        if edge_number < self.slide_start or edge_number > self.slide_end:
```

Comment:
```suggestion
                        if not self.slide_start <= edge_number <= self.slide_end:
```

Is syntactically easier / more intuitive.